### DISCUSSION TITLE
About 3D nii.gz visual plot

### DISCUSSION THREAD

**Discussion by MaoFuyou** [Project-MONAI/MONAI#7620] [General]
I want to know what class in the monai can plot nii.gz plot?

**Comment by KumoLiu** [comment]
Hi @MaoFuyou, please refer to this tutorial, https://github.com/Project-MONAI/tutorials/blob/main/modules/transform_visualization.ipynb
After `LoadImage`, you can get a torch tensor which can be used to visualize.

Hope it helps, thanks.
